packages/prepo-stores/src/MulticallStore.ts

```diff
@@ -42,6 +44,9 @@ export class MulticallStore {
   }
 
   call(): void {
+    const now = new Date().getTime()
+    if (this.lastCall + SEC_IN_MS > now) return
+    this.lastCall = now
```

Comment:
```suggestion
    if (this.lastCall > now) return
    this.lastCall = now + SEC_IN_MS
```

Maybe this way we can keep things consistent in both places we assign `lastCall`